codes.md

```diff
@@ -352,6 +369,190 @@ reasoning to the given organization that they cannot infer.
+    - Note for many of those not listed here, it's because the model became
+      more refined. See the related code.
+  - Several mentioned that aspects of their mental model were obvious
+    - [Session 009](https://osf.io/es7w6/) - "It's obviously a folder
+      structure."
+    - [Session 016](https://osf.io/d4efp/) - "You know, obviously, as graphically, with the bars, but with the description, that just helped me in my brain categorizing it better"
+    - [Session 032](https://osf.io/bka6e/) - "But I feel like in my head it’s the simplest conclusion."
+    - [Session 033](https://osf.io/uj34p/) - "Yeah so obviously the text files are the ones with the two squigglys."
+
+## Ambiguity between trees & hierarchies & sets and their common visual idioms
+  - It was difficult to determine if the participant's mental model was what
+    common dataset typologies would categorize as a tree versus a hierarchy
+versus a set. This may suggest a continuum. It may also suggest extra care be taken
+when dealing with any of these. This might suggest something about data that
+has relations in general.
+  - We are trying best-effort categorization, but the difficulty in doing this
+    supports the code:
+    - Hierarchy as Node-Link:
+	- [Session 006](https://osf.io/pkqsm/) - describes folders using the word
+	  "level", draws node-link tree
+	  - "Under that I kind of, in my head, picture text files just on an upper level, as being a little square with some writing on it or something."
+	  - "So I’m mentally, in that first sentence (draws upper level), considering the most upper-level folder, and my brain is just picturing a window with these two in them. "
+    - Hierarchy as Indented Tree/List or Database Tables or "entity
+      relationship diagram":
+      - [Session 030](https://osf.io/9jxy3/) - uses "hierarchy" and "levels"
+        for first drawing then starts talking about "normal relationship", "properties"," ownership" and
+"membership" for the database tables
+        - "I wouldn’t design, for example, if this is for an application and I had top-level code, I wouldn’t have a parallel folder that didn’t have code in it. This would be somewhere under this hierarchy (pointing to the right folder and subfolder) and referenced by whatever program was in here"
+        - "youspend a good bit of time kind of clicking from one folder to another, going up,going down, the hierarchy ladder of the stuff."
+        - "The other thing that comes to mind, if I were to try to map this out in some sort of normal relationship, then I might think of having a folders table, a files table, and the file table that has the property that tells what type of fileit is, and then I have an ownership or membership table that says, “this is inside that, and that, like that” (gesturing in space with flat hands)."
+        - "I might call that an entity relationship diagram."
+          - This might be a network at this point KATY/ALEX take second look
```

Comment:
I support the idea to call it a network. The participants hand movements as they were talking (from my memory of the interview) definitely seemed like they were connecting and linking the items/tables together.

Reply:
Will update the classification to Hierarchy evolves into Network.